### PR TITLE
[enterprise-4.12] add CoreDNS bump updates

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.12

Issue:

- https://issues.redhat.com/browse/OSDOCS-4269
- https://issues.redhat.com/browse/OSDOCS-4270
- https://issues.redhat.com/browse/OSDOCS-4342

Link to docs preview: http://file.rdu.redhat.com/jdohmann/core-dns-RN-entries/release_notes/ocp-4-12-release-notes.html#ocp-4-12-ne-coredns-bump

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@ShudiLi could you PTAL for QE ack? This PR adds an OCP 4.12 RN entry for CoreDNS version bump changes. Thank you

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -405,6 +405,15 @@ With {product-title} 4.12, a new interface has been added to the `nodeip-configu
 
 For more information, see xref:../support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues[Optional: Overriding the default node IP selection logic]. 
 
+[id="ocp-4-12-ne-coredns-bump"]
+==== CoreDNS update to version 1.10.0
+
+In {product-title} {product-version}, CoreDNS uses version 1.10.0 which includes the following changes:
```
Comment (jeana-redhat) [inline release_notes/ocp-4-12-release-notes.adoc:411]:
```suggestion
In {product-title} {product-version}, CoreDNS uses version 1.10.0, which includes the following changes:
```

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -405,6 +405,15 @@ With {product-title} 4.12, a new interface has been added to the `nodeip-configu
...
+
+* CoreDNS will not expand the query UDP buffer size if it was previously set to a smaller value.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-12-release-notes.adoc:413]:
```suggestion
* CoreDNS does not expand the query UDP buffer size if it was previously set to a smaller value.
```

File: release_notes/ocp-4-12-release-notes.adoc [hunk 3]
```diff
@@ -405,6 +405,15 @@ With {product-title} 4.12, a new interface has been added to the `nodeip-configu
...
+* CoreDNS will now always prefix each log line in Kubernetes client logs with the associated log level.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-12-release-notes.adoc:414]:
```suggestion
* CoreDNS now always prefixes each log line in Kubernetes client logs with the associated log level.
```

**Review by jeana-redhat — COMMENTED** [review]
li'l nits :)

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by ShudiLi** [comment]
/lgtm
@jldohmann thanks

**Pull request merged**